LeetCode/MinStack.cpp

```diff
@@ -0,0 +1,40 @@
+#include<stack>
+#include<iostream>
+using namespace std;
+
+class MinStack {
+public:
+    MinStack() {
+        st = stack<pair<int, int>>();
+    }
+
+    void push(int val) {
+        if (st.empty()) {
+            st.push({ val,val });
+        }
+        else {
+            st.push({ val,min(val,st.top().second) });
+        }
+    }
+
```

Comment:
We can also add peek( ) function.
void peek( ){
if(st.size( )){
return st.top( );
}
return -1;
}